test_security/test/test_secure_publisher.cpp

```diff
@@ -65,10 +65,11 @@ int main(int argc, char ** argv)
       "pass a message type\n");
     return 1;
   }
-  rclcpp::init(argc, argv);
+  const char * args[] = {"--ros-args", "--security-context", "/publisher"};
+  rclcpp::init(3, args);
```

Comment:
@ivanpauno same about hardcoding `args` element count

Reply:
Addressed in https://github.com/ros2/system_tests/pull/406/commits/65b46f0f56437040c5feff1676e522e3b2aef5c8.